### PR TITLE
Stupid commit

### PR DESCRIPTION
**Pull request by EduardLev**

### CODE REVIEW

File: LCDClock/ViewController.m [hunk 1]
```diff
@@ -19,6 +19,10 @@ - (void)viewDidLoad {
   // Do any additional setup after loading the view, typically from a nib.
 }
```
Comment (EduardLev) [inline LCDClock/ViewController.m]: hey check this out not sure about this

Comment (EduardLev) [inline LCDClock/ViewController.m]: this is a comment

File: LCDClock/ViewController.m [hunk 2]
```diff
@@ -19,6 +19,10 @@ - (void)viewDidLoad {
...
```
Comment (EduardLev) [inline LCDClock/ViewController.m]: 1

File: LCDClock/ViewController.m [hunk 3]
```diff
@@ -19,6 +19,10 @@ - (void)viewDidLoad {
...
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+}
+
```
Comment (EduardLev) [inline LCDClock/ViewController.m]: 2

File: LCDClock/ViewController.m [hunk 4]
```diff
@@ -19,6 +19,10 @@ - (void)viewDidLoad {
   // Do any additional setup after loading the view, typically from a nib.
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
```
Comment (andynovak12) [inline LCDClock/ViewController.m]: nice!

**Pull request merged**